components/blitz/resources/omero/api/RawPixelsStore.ice

```diff
@@ -337,6 +337,16 @@ module omero {
                 idempotent IntegerIntegerArrayMap getHistogram(IntegerArray channels, int binCount, bool globalRange, omero::romio::PlaneDef plane) throws ServerError;
                 
                 /**
+                 * Find the minimum and maximum pixel values for the specified channels by iterating over a full plane.
+                 * In case of multi-z/t images only the 'middle' plane with index maxZ/2, respectively maxT/2 is taken into account.
+                 * Note: This method can currently only handle non-pyramid images,
+                 * in which case an empty map will be returned.
```

Comment:
Just noticed that doesn't make sense. Of course in case of pyramid images an empyt map is returned. Will fix that in another commit after PR testing today.